app/src/main/res/layout/fragment_send_post.xml

```diff
@@ -1,66 +1,86 @@
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="@color/white"
+    android:theme="@style/Theme.BeGreen"
+    tools:context="fragments.SendPostFragment">
 
     <ImageView
         android:id="@+id/preview"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:contentDescription="@string/here_is_the_photo_that_was_taken"
+        android:scaleType="fitStart"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        android:scaleType="fitStart"
-        android:contentDescription="@string/here_is_the_photo_that_was_taken" />
+        app:layout_constraintTop_toTopOf="parent" />
 
     <com.google.android.material.textfield.TextInputLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        app:layout_constraintBottom_toTopOf="@+id/post_description_layout"
-        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginHorizontal="10dp"
         android:layout_marginBottom="12dp"
-        android:layout_marginHorizontal="10dp">
+        app:layout_constraintBottom_toTopOf="@+id/post_trash_category_layout"
+        app:layout_constraintEnd_toEndOf="parent">
 
         <com.google.android.material.textfield.TextInputEditText
-            android:id="@+id/post_category"
-            android:maxLength="10"
+            android:id="@+id/post_description"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:hint="@string/category" />
+            android:hint="@string/enter_description_here"
+            android:maxLength="250" />
+
     </com.google.android.material.textfield.TextInputLayout>
 
 
     <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/post_description_layout"
+        android:id="@+id/post_trash_category_layout"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="75dp"
         android:layout_marginStart="10dp"
+        android:layout_marginEnd="75dp"
         android:layout_marginBottom="12dp"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent">
 
-        <com.google.android.material.textfield.TextInputEditText
-            android:id="@+id/post_description"
+        <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:hint="@string/enter_description_here"
-            android:maxLength="250" />
+            android:layout_height="match_parent"
+            android:orientation="horizontal">
+            
+            <com.google.android.material.textfield.TextInputEditText
```

Comment:
Why did you use a `TextInputEditText` and disabled it instead of simply using a `TextView` ?